beacon-chain/state/v1/state_trie.go

```diff
@@ -202,10 +278,10 @@ func (b *BeaconState) Copy() state.BeaconState {
 	return dst
 }
 
-// HashTreeRoot of the beacon state retrieves the Merkle root of the trie
+// HTR of the beacon state retrieves the Merkle root of the trie
 // representation of the beacon state based on the Ethereum Simple Serialize specification.
-func (b *BeaconState) HashTreeRoot(ctx context.Context) ([32]byte, error) {
-	ctx, span := trace.StartSpan(ctx, "beaconState.HashTreeRoot")
+func (b *BeaconState) HTR(ctx context.Context) ([32]byte, error) {
+	ctx, span := trace.StartSpan(ctx, "beaconState.HTR")
```

Comment:
Name changed to avoid collision with SSZ-generated `HashTreeRoot`. The generated function should not be used, but somebody might still use it by mistake. Maybe fastssz should not generate functions when it finds already existing ones?

Reply:
Dont really like this, this is pretty easy to misuse. Someone can accidentally call HashTreeRoot and they would be using the wrong one.

Reply:
You are absolutely right. Since I don't generate the SSZ file every time, I can just remove `HashTreeRoot` from the existing one.